### PR TITLE
LPS-82022 Rename Configurations

### PR DESCRIPTION
**Pull request by martamedio**

### CODE REVIEW

File: modules/apps/portal-store/portal-store-jcr/src/main/resources/content/Language_ar.properties [hunk 1]
```diff
@@ -4,7 +4,6 @@ jackrabbit-credentials-password=Jackrabbit Credentials Password (Automatic Copy)
 jackrabbit-credentials-username=Jackrabbit Credentials Username (Automatic Copy)
 jackrabbit-repository-home=Jackrabbit Repository Home (Automatic Copy)
 jackrabbit-repository-root=Jackrabbit Repository Root (Automatic Copy)
-jcr-store-configuration-name=تكوين مخزن زملاءهم (Automatic Translation)
```
Comment (JorgeFerrer) [inline modules/apps/portal-store/portal-store-jcr/src/main/resources/content/Language_ar.properties]: I'm missing Language.properties in this commit

Comment (martamedio) [reply modules/apps/portal-store/portal-store-jcr/src/main/resources/content/Language_ar.properties]: `Language.properties` has already with the right text at `master`, but the rest of the files hadn't gone through `buildLang`

Comment (JorgeFerrer) [reply modules/apps/portal-store/portal-store-jcr/src/main/resources/content/Language_ar.properties]: Got it

**Comment by JorgeFerrer** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by JorgeFerrer** [comment]
Muchas gracias Marta. Ya se la he pasado a Brian

**Comment by JorgeFerrer** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/59868. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**